packages/core/src/codewhisperer/commands/startSecurityScan.ts

```diff
@@ -542,3 +550,27 @@ function showScanCompletedNotification(total: number, scannedFiles: Set<string>)
         }
     })
 }
+
+export async function generateZipCodeScanForProject(
+    zipUtil: ZipUtil,
+    options?: {
+        includeGitDiff?: boolean
+        silent?: boolean
+    }
+): Promise<ZipMetadata> {
+    try {
+        // We assume there is at least one workspace open.
+        return await zipUtil.zipProject(
```

Comment:
whats the difference between the implementation here and the one on line 163? could you just call this function on line 163?

Reply:
Good catch, yes that was my intention. I extracted that piece to make it easier to test.